src/extension/engine/mysql-pass.ts

```diff
@@ -115,6 +115,21 @@ export class MySQLType extends AbstractServer
         });
     }
 
+    /**
+     * @param {string} sql - a SQL string
+     * @return {string} - the SQL string without comments
+     */
+    removeComments(sql: string) {
+        const quotes=/^((?:[^"`']*?(?:(?:"(?:[^"]|\\")*?(?<!\\)")|(?:'(?:[^']|\\')*?(?<!\\)')|(?:`(?:[^`]|\\`)*?(?<!\\)`)))*?[^"`']*?)/;
```

Comment:
It seems to me that unit tests would be useful. I am planning to prepare a platform for unit testing next week. What do you think about it ?

Reply:
Good idea! :)
The code is getting more and more complex so I think unit testing could be really useful, especially for the engine classes. 👍

Reply:
@EPMatt  I have configured the unit tests.  I also added the first tests.
```
git fetch origin
gir rebase origin/master
```
Please download the changes and add tests covering your functionality.

Reply:
I fixed the previous build by removing Mocha and configuring the TypeScript compiler to properly integrate with Jest.  
I added a few tests for the comments removal units, both for PostgresSQL and MySQL engines. Those checks should be enough for making sure the functionality works as expected. :)